Multisig/UI/Assets/AssetsViewModel.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AssetsViewModel.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 13.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+struct TokenBalance: Identifiable {
+    var id: String {
+        return address
+    }
+    var imageURL: URL {
+        return URL(string: "https://gnosis-safe-token-logos.s3.amazonaws.com/\(address).png")!
+    }
+    let address: String
+    let balance: String
+    let balanceUsd: String
+}
+
+class AssetsViewModel: ObservableObject {
+    @Published var balances = [TokenBalance]()
+    @Published var isLoading: Bool = true
+    @Published var errorMessage: String? = nil
+
+    private var subscribers = Set<AnyCancellable>()
+
+    init() {
+        let request = Safe.fetchRequest().selected()
```

Comment:
the selected safe should be passed as an argument to the initializer from the view itself, because we already fetch the selected safe in the `ContentView`,  I think this would work out nicely here.

Reply:
done